src/components/calcite-color-picker/calcite-color-picker.scss

```diff
@@ -91,7 +91,32 @@ $gap--large: 12px;
 .container {
   display: inline-block;
   border: 1px solid var(--calcite-ui-border-1);
-  background-color: var(--calcite-ui-foreground-1);
+}
+
+.color-field-and-slider-wrap {
+  position: relative;
+  overflow: hidden;
+}
+
+.scope {
+  @apply text--1
+    p-0
+    mb-0
+    mr-0
+    focus-base
+    w-2
+    h-2
+    absolute
+    pointer-events-none;
+
+  margin-top: -0.25rem;
+  margin-left: -0.25rem;
+  outline-offset: 10px;
```

Comment:
large outline here allows us to keep the element small. When the element overflowed it's container, on focus it was forcing a repaint of the canvas and causing a gap below the canvas element. When the size is accomplished outline there are no issues. Actual size isn't important as this element has no mouse events and is only keyboard accessible.